src/components/OntologyBrowser/store.ts

```diff
@@ -61,6 +72,47 @@ const findAllSameTerms = (
   return sameTermKeys;
 };
 
+export const generateNavTreeFormKey = (
+  phenotypes: string[],
+  selectedPhenotypeInfoTitle: string,
+): TreeNodeLight[] => {
+  if (!phenotypes.length) {
+    return [];
+  }
+
+  if (phenotypes.length === 1) {
```

Comment:
I think we should avoid mutating when possible. For example myArray.slice(0, -1) vs myArray.pop()